modules/navier_stokes/include/fvkernels/INSFVFluxKernel.h

```diff
@@ -28,18 +28,13 @@ class INSFVFluxKernel : public FVFluxKernel, public INSFVMomentumResidualObject
   virtual ~INSFVFluxKernel() = default;
 
   using FVFluxKernel::computeResidual;
-  void computeResidual(const FaceInfo &) override final {}
+  void computeResidual(const FaceInfo & fi) override final;
   using FVFluxKernel::computeJacobian;
-  void computeJacobian(const FaceInfo &) override final {}
+  void computeJacobian(const FaceInfo & fi) override final;
   using FVFluxKernel::computeResidualAndJacobian;
-  void computeResidualAndJacobian(const FaceInfo &) override final {}
+  void computeResidualAndJacobian(const FaceInfo & fi) override final;
 
 protected:
-  ADReal computeQpResidual() override final
-  {
-    mooseError("INSFVFluxKernels must implement gatherRCData and not computeQpResidual");
-  }
```

Comment:
I d rather we dont re-introduce that

the Qp abstraction isnt great for FV because of mesh refinement. We naturally created the face infos on the more refined side, I dont the "take the Qp" from the other side worked

Reply:
What he’s done is great for code reuse. If the issue is the name itself we could consider changing that in the framework and on down

Reply:
I ended up rolling with the strong residual concept.